problems/arraysWithLoops.js

```diff
@@ -25,39 +38,71 @@ function shoutWhileLoop() {}
  * @returns {Array} - ["I!", "am!", "a!", "happy!", "array!"]
  */
 
-function shoutForOfLoop() {}
+const shoutForOfLoop = (arr) => {
+  let newArray = []
+  for(let element of arr){
+    newArray.push(element + "!");
+  }
+return newArray;
+}
+  
 
 /**
  * Takes in an array and returns the sum of all values
  * @param {number[]} nums
  * @returns {number} sum
  */
 
-function sumArray() {}
+const sumArray = (nums) => {
+  let sum = 0; 
+        for(let i = 0; i < nums.length; i++ ) {
```

Comment:
Make sure you indent accordingly. No need for things in the same scope to be indented apart from each other